code/modules/chemistry/tools/food_and_drink.dm

```diff
@@ -352,6 +362,26 @@
 	proc/on_finish(mob/eater)
 		return
 
+	proc/drop_item(var/path)
+		var/obj/drop = new path
+		if(istype(drop))
+			drop.pixel_x = src.pixel_x
+			drop.pixel_y = src.pixel_y
+			var/obj/item/I = drop
+			if(istype(I))
+				var/mob/M = src.loc
+				if(istype(M))
```

Comment:
best practice for this is to use ismob(src.loc) and then cast

Reply:
doing var/mob/M = and then an istype(M) combines the isnull check with the type check. I was told that the way I implemented this was best practice.